client/shipping-label/state/actions.js

```diff
@@ -5,17 +5,31 @@ import flatten from 'lodash/flatten';
 import omit from 'lodash/omit';
 import printDocument from 'lib/utils/print-document';
 import * as NoticeActions from 'state/notices/actions';
-
+import getFormErrors from 'shipping-label/state/selectors/errors';
+import { hasNonEmptyLeaves } from 'lib/utils/tree';
+import validateAddress from './validate-address';
 export const OPEN_PRINTING_FLOW = 'OPEN_PRINTING_FLOW';
 export const EXIT_PRINTING_FLOW = 'EXIT_PRINTING_FLOW';
 export const UPDATE_ADDRESS_VALUE = 'UPDATE_ADDRESS_VALUE';
+export const ADDRESS_VALIDATION_IN_PROGRESS = 'ADDRESS_VALIDATION_IN_PROGRESS';
+export const ADDRESS_VALIDATION_COMPLETED = 'ADDRESS_VALIDATION_COMPLETED';
+export const PICK_NORMALIZED_ADDRESS = 'PICK_NORMALIZED_ADDRESS';
+export const EDIT_ORIGINAL_ADDRESS = 'EDIT_ORIGINAL_ADDRESS';
```

Comment:
Instead of `ORIGINAL_` which could be confused with `ORIGIN` could we just say `EDIT_ADDRESS` ?